contribute/style-guides/e2e-playwright.md

```diff
@@ -0,0 +1,299 @@
+# End-to-end tests
+
+Grafana Labs uses a minimal [homegrown solution](../../e2e/utils/index.ts) built on top of [Playwright](https://playwright.dev/) for its end-to-end (E2E) tests.
```

Comment:
this should probably link to the [plugin-e2e package](https://github.com/grafana/plugin-tools/tree/main/packages/plugin-e2e) instead? 🤔